app/controllers/observations_controller/index.rb

```diff
@@ -204,11 +204,22 @@ def filtered_index_final_hook(query, _display_opts)
       query
     end
 
-    def index_display_opts(opts, _query)
+    def index_display_opts(opts, query)
       # We always want cached matrix boxes for observations if possible.
       # cache: true  will batch load the includes only for fragments not cached.
-      { matrix: true, cache: true,
-        include: observation_index_includes }.merge(opts)
+      opts = {
+        matrix: true, cache: true,
+        include: observation_index_includes
+      }.merge(opts)
+
+      # Paginate by letter if sorting by user or name.
+      if %w[user reverse_user name reverse_name].include?(
+        query.params[:order_by]
+      )
+        opts[:letters] = true
+      end
+
+      opts
```

Comment:
I'd vote to reenable alpha sorting for Species Lists and not Observations. In that case this file (only) would not get reverted. 

That would answer Terri's request.

Reply:
Revising this recommendation to accommodate Ron Pastorino's note. 

Will amend this PR accordingly.